lib/matplotlib/tests/test_datetime.py

```diff
@@ -395,3 +402,8 @@ def test_vlines(self):
     def test_xcorr(self):
         fig, ax = plt.subplots()
         ax.xcorr(...)
+
+instance = TestDatetimePlotting()
+
+
+instance.test_boxplot()
```

Comment:
```suggestion
```

This seems to have been put in for testing purposes

Reply:
yes this i forgot to remove